src/hiveboard_bridge/src/main.cpp

```diff
@@ -38,12 +58,34 @@ int main(int argc, char** argv) {
     ROS_INFO("Client connected.");
 
     HiveMindHostDeserializer deserializer(tcpServer);
+    HiveMindHostSerializer serializer(tcpServer);
     MessageHandler messageHandler;
 
+    // Register callbacks
+    CallbackFunction moveByCallback = [&](CallbackArgs args, int argsLength) {
+        swarmus_ros_navigation::MoveByMessage moveByMessage;
+
+        moveByMessage.distance_x = std::get<float>(args[0].getArgument());
+        moveByMessage.distance_y = std::get<int64_t>(args[1].getArgument());
```

Comment:
Is this temporary? thoses lines will throw if the param don't match the types

Reply:
I feel like we _should_ throw here. If the types do not match, there will be problems. 

I'll change the second param for a float for consistency, but what do you think?

Reply:
Yea good point, it will make it easier to debug